include/click/radiotap_iter.h

```diff
@@ -0,0 +1,94 @@
+#ifndef CLICK_RADIOTAP_ITER_H
+#define CLICK_RADIOTAP_ITER_H
```

Comment:
The `extern C` wrapping should go *in the header file*, I think,  as it does in click/md5.h and click/crc32.h, rather than in the including files